app/views/handlers/feedback.py

```diff
@@ -105,8 +104,8 @@ def handle_post(self) -> None:
             feedback_type=self.form.data.get("feedback-type"),
         )
 
-        message = json_dumps(feedback_message())
-
+        message = feedback_message()
+        message.update(feedback_metadata())
```

Comment:
`feedback_metadata()` should be assigned to a var and reused. It is being created twice at the moment, once here and once on upload.